guides/common/assembly_provisioning-cloud-instances-azure.adoc

```diff
@@ -1,4 +1,7 @@
-ifdef::context[:parent-context: {context}]
+:parent-context: {context}
+:context: azure-provisioning
```

Comment:
@maximiliankolb Max, we didn't realize that changing the context will mess up our links! I think we shouldn't change the context at all in these places and we should only define the additional attributes such as `:azure-provisioning:`, which will not mess up the context in links. We don't use the context attribute to condition the provisioning content anyway.

Reply:
I will file a PR to fix that.